aexpect/client.py

```diff
@@ -259,7 +259,8 @@ def _get_aexpect_helper(self, helper_cmd, pass_fds, echo, command):
             full_output += output
             sub_status = sub.poll()
             if sub_status is not None:
-                raise ExpectProcessTerminatedError(pattern, sub_status, full_output)
+                raise ExpectProcessTerminatedError(
+                    pattern, sub_status, full_output)
```

Comment:
Was this change intentional? I think this is incompatible with black linting and doesn't relate to the topic of this pull request.

Reply:
Oh I am not sure how did this happen @pevogam , I will remove this too.